tests/test_validation.py

```diff
@@ -176,6 +176,13 @@ def test_user_agent(self):
     def test_accept_language(self):
         self.check_str_type('device', 'accept_language')
 
+    def test_session_id(self):
+        self.check_str_type('device', 'session_id')
+
+    def test_session_age(self):
+        self.check_transaction({'device': {'ip_address': '4.4.4.4', 'session_age': 3600.5}})
+        for invalid in (3600, 0, -1):
```

Comment:
Why are 3600 and 0 invalid? Let's fix that so they aren't and maybe add a test for a non-numeric value too, e.g., `'foo'`.